CHANGELOG.md

```diff
@@ -1,3 +1,105 @@
+# dbt_hubspot v0.23.0
+[PR #157](https://github.com/fivetran/dbt_hubspot/pull/157) includes the following updates:
+
+## Breaking Changes
+- These updates are considered breaking due to changes in model schemas, including the addition of new columns and joins.
+
+### New Models
+- `int_hubspot__owners_enhanced`: Enriches owner records with team and role metadata
+- `int_hubspot__form_metrics__by_contact`: Adds form submission and conversion metrics at the contact level
+- `hubspot__engagement_communication`: Represents the relationship between communications and engagements
+
+### Updated Models
+The following models have been updated with new columns sourced from the new intermediate models:
+
+- `int_hubspot__owners_enhanced` is joined to:
+  - `hubspot__deals`
+  - `hubspot__tickets`
```

Comment:
request to link to the model docs page (and for the other models that had new columns added)

Reply:
Added.